### DISCUSSION TITLE
jooby 4: next major release/wish list

### DISCUSSION THREAD

**Discussion by jknack** [jooby-project/jooby#3547] [Ideas]
What would you expect in next major release?

**Comment by SentryMan** [comment]
It'd be neat if we could have the config framework be pluggable.

**Comment by kliushnichenko** [comment]
Please consider using [ProblemDetailsHandler](https://jooby.io/#problem-details) (RFC7807) as the default error handler.
I believe it's a good strategy to follow widely known and adopted standards.
Making it the default should also make things cleaner and enable smoother integration with other modules.

**Reply by kliushnichenko** [reply]
Hi @jknack. 
I see v4 has been released. I was wondering if this item is still on your radar, or if you’ve decided to keep it as it is

**Reply by jknack** [reply]
Hi @kliushnichenko, yea I kept as it is. Same for configuration.

**Comment by jknack** [comment]
should 4.x require Java 21?
